gateway/v10.ts

```diff
@@ -2213,6 +2215,61 @@ export interface GatewayMessagePollVoteDispatchData {
 	answer_id: number;
 }
 
+/**
+ * @see {@link https://discord.com/developers/docs/events/gateway-events#rate-limited}
+ */
+export type GatewayRateLimitedDispatch<
```

Comment:
This should be a union of concretes, not require users to pass in the op code they want imo. That said, how you'd present this is...gonna be fun

Reply:
I made it as close as docs as possible.
Additionally, the type parameter has a default (all the valid opcodes, which is only one anyways), if that's your concern

Reply:
Done, assuming I understood what you meant a59d659bf7b4b4f3de89cd0160bd24bbf4ec672a